quickstarts/Audio.ipynb

```diff
@@ -17,27 +17,37 @@
     },
```

Comment:
<p>Can we just say "Attach it to the prompt in the list of parts." (omitting the glm.Blob detail, I don't see that mentioned in the code - looks like it works automagically). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
And, thank you for adding this!

Reply:
<p>Done.</p>